engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -116,6 +130,23 @@ public void testFactorial() throws Exception {
     """);
   }
 
+  @Test
+  public void testIsDigitWithSpaces() throws Exception {
+    parseTest("""
+    compare =
+        is_digit = character -> 42
+    """);
+  }
+
+  @Test
+  @Ignore
```

Comment:
@kazcw, this test require changes on the Rust side. Presumably the `Tree` shall be identical to `testIsDigitWithSpaces`.